server/task.go

```diff
@@ -558,8 +558,7 @@ func (s *Server) changeTaskStatusWithPatch(ctx context.Context, task *api.Task,
 	// 1. Assign the proper project to the newly created database. Otherwise, the periodic schema
```

Comment:
Let's update the comment. If code below is completely removed below, we should delete the comment.